tests/kafkatest/services/connect.py

```diff
@@ -39,6 +40,16 @@ class ConnectServiceBase(KafkaPathResolverMixin, Service):
     STDERR_FILE = os.path.join(PERSISTENT_ROOT, "connect.stderr")
     LOG4J_CONFIG_FILE = os.path.join(PERSISTENT_ROOT, "connect-log4j.properties")
     PID_FILE = os.path.join(PERSISTENT_ROOT, "connect.pid")
+    CONNECT_REST_PORT = 8083
+    """
```

Comment:
I think this docstring isn't applying to what you think it is.

```
String literals occurring immediately after a simple assignment at the top level of a module, class, or __init__ method are called "attribute docstrings".
```

This docstring is applying to `CONNECT_REST_PORT` rather than the subsequent `STARTUP_MODE_*` constants. This is unfortunate about not having enums in py2. You could either document each separately, or put them under a nested class for namespacing.

Reply:
:) Gotcha